include/smac_planner/utils.hpp

```diff
@@ -53,6 +52,35 @@ class Utils {
     return msg;
   }
 
+  /**
+  * @brief Check if the position of given x,y point is below or above the pose
+  * @param x float of X coordinate
+  * @param y float of Y coordinate
+  * @param reference_pose the pose with respect to which we want to check the position of the x,y point
+  * @return Bool true means point is below pose, false means point is above pose
+  */
+  static inline bool isBehindPose(
+    const geometry_msgs::Pose & pose,
+    const geometry_msgs::PoseStamped & reference_pose){
```

Comment:
```suggestion
  static inline bool isBehindPose(
    const geometry_msgs::Point & point,
    const geometry_msgs::Pose & reference_pose){
```
In pose, you don't need the orientation, then it is `Point`
In reference_pose you don't need header, then it is `Pose`
Keep the signature to the minimal